dhctl/pkg/operations/mirror/validate.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 Flant JSC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mirror
+
+import (
+	"fmt"
+
+	"github.com/google/go-containerregistry/pkg/v1/layout"
+	"github.com/google/go-containerregistry/pkg/v1/validate"
+)
+
+type ValidationMode string
+
+const (
```

Comment:
duplicated with it. Need to keep only one source of true 
https://github.com/deckhouse/deckhouse/pull/6257/files#diff-0596f6ae70c0bb6f7ee4ecb270c0d20ab22e959a76cf213b4dd0b2f25409d889R34